package.json

```diff
@@ -6,8 +6,10 @@
     "start": "webpack-dev-server --mode=development",
     "build": "webpack --mode=production",
     "lint": "eslint --ext js,jsx src",
-    "test": "jest",
-    "ci": "npm run lint && npm run build"
+    "test": "jest src --env=jsdom",
+    "test:cover": "jest src --coverage",
+    "ci": "npm run lint && npm run build",
```

Comment:
As we implemented tests we need to include them to our CI process, so please add the `npm run test` to the `ci` script.